deeplinkdispatch-processor/build.gradle

```diff
@@ -11,8 +12,9 @@ dependencies {
   implementation deps.jsr305
   implementation deps.kotlinStdLib
   implementation deps.javaPoet
-  implementation deps.autoCommon
   implementation deps.androidXAnnotations
+  implementation deps.ksp
+  implementation deps.xProcessor
```

Comment:
How are you thinking we should handle the fact that xProcessor is still in alpha, and ksp in beta for the public release?

Reply:
We could release a DLD next major version beta.

Reply:
We can drop the feature in the next release For this one it does not make that much sense dropping it as itis all working already. And yes this tests incremental compilation.